rmf_demos_dashboard_resources/airport_terminal/app-config.json

```diff
@@ -1,9 +1,8 @@
 {
+  "$schema": "./app-config.schema.json",
```

Comment:
```suggestion
  "$schema": "https://raw.githubusercontent.com/open-rmf/rmf-web/main/packages/dashboard/app-config.schema.json"
```

Reply:
hmm, I guess we might still change the location of where the config should be copied too, yeah an absolute url makes more sense, thanks for the suggestion, https://github.com/open-rmf/rmf_demos/pull/257/commits/b3ef3b002b58077a92f201ee4783d465fa0f79da